e2e/protractor/protractor.excludes.json

```diff
@@ -6,13 +6,7 @@
   "C286252": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
   "C284666": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
   "C286269": "temp, see https://alfresco.atlassian.net/browse/ACS-5189",
-  "C279191": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
-  "C279192": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
-  "C279193": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
-  "C279195": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
   "C280051": "temp, will be fixed in https://alfresco.atlassian.net/browse/ACS-5183",
```

Comment:
@MichalKinas some of e2es can't be fixed without changes for your jira. Can we fix them later?

Reply:
Yes no worries

Reply:
Let's just add some comment or task for it not to forget about it

Reply:
@MichalKinas thanks, I just raised jira for that and corrected link in excluded list to indicate on that jira 

https://alfresco.atlassian.net/browse/ACS-5632